frontend/README.md

```
@@ -0,0 +1,12 @@
+# Gleam frontend!
+
+This is the Lumina frontend, written in Gleam.
+
+## Why the externals?
+
+The Gleam frontend is written in Gleam, but it uses a lot of JavaScript. This is because the Gleam ecosystem is still young and the existing libraries do not cover all the functionality we need. We hope that in the future we can replace the JavaScript with Gleam code.
+
+(Also, I was kinda lazy and didn't want to be translating JavaScript to Gleam all day.)
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)